entries/301_show_notes.bacf

```diff
@@ -0,0 +1,72 @@
+- nr: 301
+- title: Live from CocoaHeads Hamburg
+- date: 14.01.2020, 20:00
```

Comment:
Plus I put this to Tuesday, but that can be changed depending on when we want to release it of course.